ooniapi/services/oonifindings/src/oonifindings/routers/v1.py

```diff
@@ -131,55 +143,60 @@ def list_oonifindings(
     """
     log.debug("listing incidents")
-    where = "WHERE deleted != 1"
-    query_params = {}
+    q = db.query(models.OONIFinding).filter(
+        models.OONIFinding.deleted != 1
+    )
 
+    client_role = get_client_role(authorization, jwt_encryption_key=settings.jwt_encryption_key)
     account_id = get_account_id_or_none(
         authorization, jwt_encryption_key=settings.jwt_encryption_key
     )
+
     if only_mine:
-        if account_id is None:
-            return OONIFindingIncidents(incidents=[])
-        where += "\nAND creator_account_id = %(account_id)s"
+        q = q.filter(models.OONIFinding.creator_account_id == account_id)
 
     if account_id is None:
         # non-published incidents are not exposed to anon users
-        where += "\nAND published = 1"
-        query_params["account_id"] = "never-match"
-    else:
-        query_params["account_id"] = account_id
-
-    query = f"""SELECT id, update_time, start_time, end_time, reported_by,
-    title, event_type, published, CCs, ASNs, domains, tags, test_names,
-    links, short_description, email_address, create_time, 
-    creator_account_id = %(account_id)s as mine
-    FROM incidents FINAL
-    {where}
-    ORDER BY title
-    """
-    q = query_click(db=db, query=query, query_params=query_params)
-    
-    incidents = list(q) 
-    client_role = get_client_role(authorization, jwt_encryption_key=settings.jwt_encryption_key)
-    for incident in incidents:
-        incident["published"] = bool(incident["published"])  
+        q = q.filter(models.OONIFinding.published == 1)
+        account_id = "never-match"
+
+    findings = []
+    for row in q.all():
+        oonifinding = OONIFinding(
+            id=row.incident_id,
+            update_time=row.update_time,
+            start_time=row.start_time,
+            end_time=row.end_time,
+            reported_by=row.reported_by,
+            title=row.title,
+            event_type=row.event_type,
+            published=bool(row.published),
+            CCs=row.CCs,
+            ASNs=row.ASNs,
+            domains=row.domains,
+            tags=row.tags,
+            test_names=row.test_names,
+            links=row.links,
+            short_description=row.short_description,
+            email_address=row.email_address,
+            create_time=row.create_time,
+            mine=(row.creator_account_id == account_id)
+        )
+
         if account_id is None or client_role != "admin":
-            incident["email_address"] = ""
-    
+            oonifinding.email_address = ""
+
+        findings.append(oonifinding)
+
     setnocacheresponse(response)
-    incident_models = []
-    for i in range(len(incidents)):
-        incident = incidents[i]
-        incident_model = OONIFinding.model_validate(incident)
-        incident_models.append(incident_model)
-    return OONIFindingIncidents(incidents=incident_models)
+    return OONIFindingIncidents(incidents=findings)
 
 
 @router.get(
```

Comment:
Same here, there is no need in RESTful API design to include the verb in the path, it can just be `/v2/finding/{finding_id | slug}`